### ISSUE TITLE
Supply resource missing

### ISSUE THREAD

**Issue by Worm61** [BobPalmer/USI-LS#122]
I recently started playing KSP again after a long time away. I've been away for a few updates, so today I reinstalled all my USI mods. Now when I add life support tanks to my ships, they still show as having no supplies on the Life Support menu. I thought maybe the containers defaulted as empty, so I tried to fill them in the SPH, but I couldn't open up the menu. I right-click on the tank and nothing happens. This is happening with every life support container I've tried. The effects of the mod continue to work, so the Kerbals act as if they have no supplies, and the countdown to mutiny starts.

**Comment by BobPalmer** [comment]
Bad install, you are missing CRP.

**Comment by Worm61** [comment]
With the issue closed, I'm not sure this post will be visible, so I made a new thread as well.

You closed the post without giving me a chance to reply. I have downloaded both the Life Support mod and the separate Resource Pack at least three times each. The supply resource, as well as a number of others, is missing every single time I download and install it. I looked through the folders, and the resource folder is there, and has several resources in it, but it is also noticeably missing some as well. If it's a bad install, it seems impossible for me to get a good one. What do I do about that?
